.circleci/docker/common/install_conda.sh

```diff
@@ -90,6 +90,8 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
     conda_install magma-cuda110 -c pytorch
   fi
 
+  conda_install valgrind -c conda-forge
```

Comment:
Valgrind from conda, not from apt? (Also, is `conda-forge` dragging in packages it shouldn't?)

Reply:
I think this made more sense when we also needed headers and conda provided both. Probably obsolete now.

Reply:
Switched from conda to apt/yum. I'll mark resolved if CI works.